testing/lsan_suppressions.txt

```diff
@@ -40,3 +40,25 @@ leak:flutter::Engine::Run(flutter::RunConfiguration)
 leak:txt::FontCollection::CreateMinikinFontFamily
 leak:CGFontCreateFontsWithURL
 leak:txt::GetDefaultFontFamily
+
+# The RefCounted DebugChecks tests intentionally fail to wrap and manage heap
+# allocated objects.
+leak:RefCountedTest_DebugChecks_Test::TestBody
+
+# Objects allocated by this mock API are intentionally not collected and are
+# instead marked as released in order to explicitly assert double free attempts
+# and allow some tests to inspect contents.
+leak:*flutter/shell/platform/linux/testing/mock_engine.cc
+
+# TODO(bdero): Fix FL leaks
```

Comment:
Please file a new issue or make a note in an existing issue, and add a link to the issue to this TODO comment.

Reply:
Good idea, done: https://github.com/flutter/flutter/issues/90155